src/rockstor/smart_manager/data_collector.py

```diff
@@ -428,7 +428,13 @@ def logs_downloader(logs_queued, recipient):
                 download_command.append(self.build_log_path(log))
 
             # Build download archive
-            download_process = Popen(download_command, bufsize=1, stdout=PIPE)
+            download_process = Popen(
```

Comment:
@phillxnet , @Hooverdan96 ,
Note that this change is not required _per se_... even without them, one can still downloads the tgz of the logs and read the files therein without issue. I've added the change here nonetheless for consistency more than anything.
I can remove that one if you feel it's a bad idea.

Reply:
my 2 cents: consistency is good, and seems to me this is a small change (and risk) to get things in line ...

Reply:
Agreed - consistency is good.

Reply:
Thank you both for the feedback!